tasks_week3_Israel_Jacob/include/Location.h

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include <iostream>
+struct Location
+{
+	int row;
+	int colomn;
+public:
+	Location();
+	Location(int row, int colomn);
+	int getRow();
```

Comment:
no need for getters, this is not a class

Reply:
Got deleted